ouroboros-consensus/src/Ouroboros/Consensus/ChainSyncClient.hs

```diff
@@ -195,7 +196,30 @@ chainSyncClient
        , ProtocolLedgerView blk
        , Exception (ChainSyncClientException blk tip)
        )
-    => Tracer m (TraceChainSyncClientEvent blk tip)
+    => (tip -> WithOrigin SlotNo)
+    -> Tracer m (TraceChainSyncClientEvent blk tip)
+    -> NodeConfig (BlockProtocol blk)
+    -> BlockchainTime m
+    -> ClockSkew                                    -- ^ Maximum clock skew
+    -> STM m (ExtLedgerState blk)                   -- ^ Get the current ledger state
+    -> STM m (HeaderHash blk -> Bool, Fingerprint)  -- ^ Get the invalid block checker
+    -> StrictTVar m (CandidateState blk)            -- ^ Our peer's state var
+    -> AnchoredFragment (Header blk)                -- ^ The current chain
+    -> Consensus ChainSyncClientPipelined blk tip m
+chainSyncClient = chainSyncClient_ $ pipelineDecisionLowHighMark 200 300
```

Comment:
Add a todo to make the low / high marks configurable.  It might be good to experiment a bit with the parameters and tune them to get the best performance.

Reply:
See f76f2f7. I think the networking team or QA are the right people to tune the parameters for best performance.